src/legacy/ui/public/kbn_top_nav/kbn_top_nav.js

```diff
@@ -17,108 +17,15 @@
  * under the License.
  */
 
-import 'ngreact';
-import { wrapInI18nContext } from 'ui/i18n';
 import { uiModules } from 'ui/modules';
 import { npStart } from 'ui/new_platform';
+import {
```

Comment:
Lets move the directive definition to `kibana_legacy` and update the imports from the 4 places that consume those directives (maps, dashboard_mode, timelion and kibana).

Then we'll be able to remove this folder altogether and simplify the code.